### PR TITLE
Parse CORS origins from .txt file.

### PR DESCRIPTION
**Pull request by lukaDoric**

### CODE REVIEW

File: SmartTutor/Startup.cs [hunk 1]
```diff
@@ -171,5 +171,17 @@ private static string CreateConnectionStringFromEnvironment()
             return
                 $"Server={server};Port={port};Database={database};User ID={user};Password={password};Integrated Security={integratedSecurity};Pooling={pooling};";
         }
+
+        private static string[] ParseCorsOrigins()
+        {
+            string[] corsOrigins = { "http://localhost:4200" };
```
Comment (DanijelRadakovic) [inline SmartTutor/Startup.cs:177]: I get the idea to allow front-end web app access by default, but fixing this in code is not a good practice and not secure either. My proposal is to remove this line of code and put URL in cors.txt file. This should be removed and put in cors.txt file. Setup.md file should be changed also by adding some instructions how to setup CORS.

Comment (DanijelRadakovic) [reply SmartTutor/Startup.cs:177]: I am also suggesting creation of configuration file or files for Smart Tutor service in order to remove all magical URLs, usernames, ports etc. from source code and reduce the number of environment variables. In that way, separation between source code and its configuration would be much clearer.

**Review by DanijelRadakovic — APPROVED** [review]

File: SmartTutor/Startup.cs (hunk 1 above)
Comment (Luburic) [reply SmartTutor/Startup.cs:177]: Yeah, we should move this to configuration definitely.

**Review by Luburic — APPROVED** [review]
Comment inline. We should move the CORS configuration

**Pull request merged**